webapp/components/Panel.js

```diff
@@ -26,42 +48,19 @@ export default class extends PureComponent {
 
   // a method to create the routes
   // returns the view Component with props
-  childRoute(Component, name = '') {
+  childRoute(Component, name = '', pathProps) {
     // get props needed for each route
     // 'name' should correspond with the plugin name for each route
     const routeProps = getRouteProps(name, this.props);
 
-    return <Component {...routeProps} />;
-  }
-
-  UNSAFE_componentWillMount() {
```

Comment:
This is just cleanup. Was able to remove this method that's getting deprecated

Reply:
👍